spec/Dynamic-Validator.md

```diff
@@ -265,7 +265,7 @@ fn reprioritize(candidates: &mut Vec<Candidate>, target: &Address) {
 }
 ```
 
-### on TermEnd events
+On the `TermEnd` event, the candidates will be reordered with the following algorithm.
```

Comment:
The `TermEnd` is not a subsentence of 'Candidate prioritizing'. It is parallel to them.